src/base/system_parameter.c

```diff
@@ -2366,7 +2366,8 @@ static UINT64 prm_ddl_audit_log_size_upper = 2147483648ULL;	/* 2G */
 static unsigned int prm_ddl_audit_log_size_flag = 0;
 
 char *PRM_PAGE_SERVER_HOST = NULL;
-static char *prm_page_server_host_default = "";
+static char prm_page_server_host_default_impl_arr[] = "";
+static char *prm_page_server_host_default = (char *) prm_page_server_host_default_impl_arr;
```

Comment:
to remove a g++ compiler warning

Reply:
what was the warning?

Reply:
```
server.c:316:38: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
   constexpr char *SERVER_TYPE_LONG = "type";
```

Reply:
What do you think of the next solution to the warning?
```suggestion
static const char *prm_page_server_host_default = "";
```